docs/io.cozy.files_metadata.md

```diff
@@ -1,5 +1,9 @@
 # Metadata on `io.cozy.files` documents
 
+## Generic metadata
+
+For generic metadata like `createdAt` or `updatedAt`, please follow the [cozy-doctypes doc](https://github.com/cozy/cozy-doctypes/#generic-model) about generic model
```

Comment:
We don't use absolute path since this documentation is also published on docs.cozy.io . And if we read the doc from there and then click on a link to github.com we leave our doc's website. 

To fix that, you can use relative link: 
`
./#generic-model
`

should do the trick, both working on github.com but also on docs.cozy.io